src/modules/number/index.ts

```diff
@@ -84,22 +85,26 @@ export class NumberModule extends SimpleModuleBase {
   }
 
   /**
-   * Returns a single random floating-point number for a given precision or range and precision.
-   * The lower bound is inclusive, the upper bound is exclusive, unless precision is passed.
+   * Returns a single random floating-point number for a given multipleOf or range and multipleOf.
```

Comment:
This parts sounds off to me.

Maybe use just:

```suggestion
   * Returns a single random floating-point number.
```